[FEAT] hub integration with dataset and configuration (#5161)
recognai/rubrix: argilla/src/argilla/datasets/_export/card/argilla_template.md

---
# For reference on model card metadata, see the spec: https://github.com/huggingface/hub-docs/blob/main/datasetcard.md?plain=1
# Doc / guide: https://huggingface.co/docs/hub/datasets-cards
{{ card_data }}
---

# Dataset Card for {{ repo_id.split("/")[-1] }}

{% if homepage_url %}
- **Homepage:** [{{ homepage_url }}]({{ homepage_url }})
{% endif %}
{% if repo_url %}
- **Repository:** [{{ repo_url }}]({{ repo_url }})
{% endif %}
{% if paper_url %}
- **Paper:** [{{ paper_url }}]({{ paper_url }})
{% endif %}
{% if leaderboard_url %}
- **Leaderboard:** [{{ leaderboard_url }}]({{ leaderboard_url }})
{% endif %}
{% if point_of_contact %}
- **Point of Contact:** {{ point_of_contact }}
{% endif %}

> This dataset has been created with [Argilla](https://argilla-io.github.io). As shown in the sections below, this dataset can be loaded into Argilla as explained in [Load with Argilla](#load-with-argilla), or used directly with the `datasets` library in [Load with `datasets`](#load-with-datasets).

Reviewer: ```suggestion
This dataset has been created with [Argilla](https://argilla-io.github.io). As shown in the sections below, this dataset can be loaded into Argilla as explained in [Using this dataset with Argilla](#using-this-dataset-with-argilla), or used directly with the `datasets` library in [Using this dataset with datasets](#using-this-dataset-with-datasets).
```

> This dataset has been created with [Argilla](https://argilla-io.github.io). As shown in the sections below, this dataset can be loaded into Argilla as explained in [Load with Argilla](#load-with-argilla), or used directly with the `datasets` library in [Load with `datasets`](#load-with-datasets).

Reviewer: The link doesn't seem to work, should we use the github link as an entry point (there's an overview, link to docs, and we might even get a star?)

> This dataset has been created with [Argilla](https://argilla-io.github.io). As shown in the sections below, this dataset can be loaded into Argilla as explained in [Load with Argilla](#load-with-argilla), or used directly with the `datasets` library in [Load with `datasets`](#load-with-datasets).

Reviewer: ```suggestion
This dataset has been created with [Argilla](https://argilla-io.github.io). As shown in the sections below, this dataset can be loaded into your Argilla server as explained in [Load with Argilla](#load-with-argilla), or used directly with the `datasets` library in [Load with `datasets`](#load-with-datasets).
```

## Using this dataset with Argilla

To load with Argilla, you'll just need to install Argilla as `pip install argilla --pre --upgrade` and then use the following code:

```python
import argilla as rg

ds = rg.Dataset.from_hub("{{ repo_id }}")
```

> This will load the settings and records from the dataset repository and push them to the Argilla instance.

Reviewer: ```suggestion
This will load the settings and records from the dataset repository and push them to you Argilla server for exploration and annotation.
```

> ---

Reviewer: All this will need to be updated before the release

Author: Yep. I'm on that today.
